apps/web/src/pages/templates/TemplatesListPage.tsx

```diff
@@ -103,11 +108,14 @@ function NotificationList() {
       <PageHeader
         title="Notification Template"
         actions={
-          <Link to="/templates/create" data-test-id="create-template-btn">
-            <Button disabled={readonly} icon={<PlusCircle />}>
-              New
-            </Button>
-          </Link>
+          <Button
+            disabled={readonly}
+            onClick={handleRedirectToCreateTemplate}
+            icon={<PlusCircle />}
+            data-test-id="create-template-btn"
+          >
+            New
+          </Button>
```

Comment:
it was a link which was still clickable even the button was disabled, but now it's a regular button with a redirect logic on click